WordPress/troubleshooting-guides/tsg_existing_wordpress_installation_detected.md

```diff
@@ -0,0 +1,86 @@
+# Troubleshooting Guide for Existing WordPress Site Detected Warning
+
+## Overview
+If you see the following warning message on your WordPress site homepage:
+
+<img src="../media/existing-site-detected.png" width="800">
+
+This guide will explain the purpose of this warning and provide steps to resolve it yourself, with important notes on potential caveats.
+
+## Why Am I Seeing This Warning?
+This warning serves as a safeguard against potential data loss, especially in cases where the same MySQL database server is mistakenly linked to multiple WordPress sites or when an incorrect backup restore could erase existing data. This warning allows you to diagnose the situation if you believe it’s a false alarm or if you’re prepared to proceed with the assumption of a fresh installation.
```

Comment:
This is to safeguard against potential data loss. However, the reason is incorrect. This issue happens when the necessary WordPress files are not detected in the file storage, but MySQL database already has WordPress installation present. 
This can happen due to multiple reasons:
1. If the WEBSITES_ENABLE_APP_SERVICE_STORAGE is set to false (i.e., the container is not mounted to remote storage.)
2. If the user has accidentally deleted the necessary files from file storage.
3. If there is some issue with underlying remote storage mounting or some failure of I/O operations. (i.e., some platform level issue) which cause the failure of detecting these files.

Instead of doing a fresh installation which might overwrite the existing MySQL data, we warn the user with this page.